vtex/omnishipping
# CHK-129: Fix mandatory delivery address

#### What is the purpose of this pull request?

This PR fixes the scenario where the user could proceed to the payment step even when the delivery address was not valid.

To do that, we are now validating the delivery address to decide if the payment button should be presented.

#### How should this be manually tested?

- Get to this [workspace](https://mandatoryaddress--vtexgame1.myvtex.com/checkout/cart/add/?sku=282&qty=1&seller=1&sc=1)
- Proceed to checkout
- `test@mailinator.com`
- Click on "Entregar em outro endereço"
- You should only see the payment button if the address is valid
 
#### Screenshots or example usage

![image](https://user-images.githubusercontent.com/12574426/86900073-41a92480-c0e1-11ea-90fa-e5bd67fc4531.png)

![image](https://user-images.githubusercontent.com/12574426/86900148-5d142f80-c0e1-11ea-83a1-c4b9373a542b.png)

#### Types of changes
